package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "ca11y",
-  "version": "0.1.3",
+  "version": "0.1.4",
   "description": "A lightweight accessible datepicker",
   "homepage": "http://code.viget.com/ca11y/",
   "main": "dist/index.js",
   "engines": {
-    "node": "0.12.2"
+    "node": "> 0.12.2"
```

Comment:
Ah okay. I'm actually not sure if NPM enforces this. It _might_. It would be important if you depended on the native `Promise` or something (which I believe is only supported >= 0.12.x). 

It's probably worth keeping. But what about something like `>= 0.12.x`?